assets/css/styles.css

```diff
@@ -12,7 +12,7 @@ margin: 0;
     margin-right: auto;
     /* padding: 20px;
     width: 800px; */
-    border: 1px solid red;
+    /* border: 1px solid red; */
 }
 
 #name,#email,#age,
```

Comment:
Hey @DebyGrey, you should consider using class-selectors for styles in your project instead of id-selectors, so when you need to implement javascript you can use the id selectors.